src/components/text-area/TextArea.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { forwardRef, useMemo, useRef, useState } from "react";
+import { Platform, TextInputProps, ViewStyle } from "react-native";
+
+import { Box, BoxProps, RNTextInput, TouchableProps } from "../../primitives";
+import { useTheme } from "../../theme";
+import {
+  cx,
+  generateBoxShadow,
+  RenderPropType,
+  runIfFn,
+  styleAdapter,
+  useOnFocus,
+  useOnHover,
+} from "../../utils";
+import { createComponent } from "../../utils/createComponent";
+import { composeEventHandlers } from "../../utils/mergeRefs";
+import { Spinner } from "../spinner";
+import { createIcon, Icon, TextAreaSuffix } from "..";
+
+export type TextAreaVariants = "outline" | "subtle" | "underline" | "ghost";
```

Comment:
We can remove `ghost` as its not present in the design system

Reply:
okay